lib/blockscore/base.rb

```diff
@@ -1,10 +1,12 @@
 require 'blockscore/connection'
+require 'uri'
 
 module BlockScore
   class Base
     extend Connection
 
     ABSTRACT_WARNING = 'Base is an abstract class, not an API resource'.freeze
+    API_URL = URI.parse('https://api.blockscore.com/').freeze
```

Comment:
How do you feel about using `Addressable::URI` instead of the built-in `URI` class? The API is cleaner, it's much better tested, and it handles cases where `URI` falls over.